src/rules/banCommaOperatorRule.ts

```diff
@@ -21,19 +21,40 @@ import * as ts from "typescript";
 import * as Lint from "../index";
 
 export class Rule extends Lint.Rules.AbstractRule {
-    /* tslint:disable:object-literal-sort-keys */
+    /* tslint:disable:object-literal-sort-keys max-line-length */
     public static metadata: Lint.IRuleMetadata = {
         ruleName: "ban-comma-operator",
-        description: "Bans the comma operator.",
+        description: "Disallows the comma operator to be used. [Read more about the comma operator here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comma_Operator).",
```

Comment:
In the rules list this text will be used as link to the rule's documentation page.
We should avoid putting a link inside another link.

Consider using `descriptionDetails` instead.

Reply:
Ah good point - I forgot about that!